src/librustc_mir/interpret/eval_context.rs

```diff
@@ -704,6 +699,12 @@ impl<'mir, 'tcx, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
         ::log_settings::settings().indentation -= 1;
         let frame = self.stack.pop().expect("tried to pop a stack frame, but there were none");
 
+        if let Some(return_place) = frame.return_place {
+            // Copy the return value to the caller's stack frame.
```

Comment:
```suggestion
        // Copy the return value to the caller's stack frame.
        if let Some(return_place) = frame.return_place {
```
This keeps the general style of top-level comments in this function.

Also, if `return_place` is `None`, this should raise UB. (`Unreachable` seems fine.)

Oh, and I think this is actually wrong to do this if we are currently unwinding.

And finally, you can remove the `validate_operand` below. The `copy` will perform the required validation.